plugins/sonarqube/src/index.ts

```diff
@@ -15,4 +15,8 @@
  */
 
 export * from './components';
-export { plugin } from './plugin';
+export {
+  sonarQubePlugin,
```

Comment:
i guess this is also something for the naming pattern discussion, if `sonarqube` all lowercase is the actual product name, just like we prefer `github` over `gitHub` nowadays